src/library/trackset/baseplaylistfeature.cpp

```diff
@@ -323,10 +355,33 @@ void BasePlaylistFeature::slotCreatePlaylist() {
     }
 }
 
+/// Returns a playlist that is a sibling inside the same parent
+/// as the start index
+int BasePlaylistFeature::selectSiblingPlaylistId(QModelIndex& start) {
```

Comment:
```suggestion
int BasePlaylistFeature::getSiblingPlaylistIdOf(QModelIndex& start) {
```
we do not select here. The Qt sibling() function verifies if the test position is a sibling. Here you look at all columns below start.

Reply:
It takes the next sibling if it exists, otherwise it checks if the one in front exists and returns an invalid id otherwise. I can understand that select could be misinterpreted in gui context.